Source/CesiumRuntime/Public/CesiumRasterOverlay.h

```diff
@@ -125,8 +137,33 @@ class CESIUMRUNTIME_API UCesiumRasterOverlay : public UActorComponent {
   UFUNCTION(BlueprintCallable, Category = "Cesium")
   void SetSubTileCacheBytes(int64 Value);
 
+  /**
+   * Activates this raster overlay, which will add it to (that is: display it
+   * on) the Cesium3DTileset to which the component is attached, if it hasn't
+   * already been added. The overlay will continue to be shown on the tileset
+   * until it is deactivated.
+   *
+   * If the overlay is already displayed on the Cesium3DTileset, calling this
+   * function will not cause it to pick up any new values for properties that
+   * have been modified since it as added. To do that, call Refresh.
+   *
+   * If you created this overlay component via Blueprints, consider setting the
+   * "Auto Activate" property to false on the "Add Component" node and call
+   * Activate after setting all the desired properties. This will avoid the need
+   * to call Refresh, and will ensure the overlay is not loaded multiple times.
+   *
+   * @param bReset  Whether the activation should happen even if ShouldActivate
+   * returns false.
+   */
   virtual void Activate(bool bReset) override;
+
+  /**
+   * Deactivates this raster overlay. This will remove it from (that is: stop
+   * displaying it on) the Cesium3DTileset to which the component is attached.
```

Comment:
```suggestion
   * Deactivates this raster overlay. This will stop displaying it on (that is:
   * remove it from) the Cesium3DTileset to which the component is attached.
```